silverscreen/settings.py

```diff
@@ -47,6 +47,7 @@
     'registration',
     'errors',
     'newsletter',
+    'csv_management',
```

Comment:
I don't see a need to make a new app for this now. This functionality is pretty much part of the reservations app as it is a download of his reservations. Plus, I don't see why we should put`/my_reservations` in an app called `csv_management` as the view just displays them on frontend with no csv implications.

And also try to use `-` instead of `_` in urls. You can read more about it online if you want. (https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris)

Maybe it would make sense to have an app called `csv_management` if your app deals with a lot of csv related logic and this structure would make your project more scalable. But this is not our case.